main.cpp

```diff
@@ -191,128 +191,330 @@ struct CarWash //                                   1) define an empty struct fo
+    int towerHeight = 13;
+    //number of fans (int)
+    int numOfFans = 3;
+    //amount of ram (GB)(int)
+    int amountOfRam = 16;
+    //number of usb ports (int)
+    int numOfUSBPorts = 6;
+    //number of keys (int)
+    int numOfKeys = 120;
+
+    //display letters typed
+    void typeLetter(bool isLetterKeyPressed);
+    //multiply numbers
+    int productOfNumbers(int num1, int num2);// returns product of numbers
+    //run software
+    void runSoftware(bool hasDoubleClickedIcon);
+};
 /*
 Thing 4) Nuclear Power Plant
 5 properties:
-    1) amount of water (float)
-    2) temp. of water (float)
+    1) amount of water used (gallons per mega-watt hr.) (float)
+    2) temp. of water (celcius) (float)
     3) number of generators (int)
-    4) weight of generator (float)
+    4) weight of generator (tons) (float)
     5) number of engineers (int)
 3 things it can do:
     1) produce heat
     2) generate steam
     3) produce electricity
  */
-
+struct NuclearPowerPlant
+{
+    //amount of water used (gallons per mega-watt hr.) (float)
+    float amountOfWaterUsed = 375.5f;
+    //temp. of water (celcius) (float)
+    float tempOfWater = 284.8f;
+    //number of generators (int)
+    int numOfGenerators = 3;
+    //weight of generator (tons) (float)
+    float weightOfGenerators = 389.79f;
+    //number of engineers (int)
+    int numOfEngineers = 100;
+
+    //produce heat
+    void produceHeat(bool isFissionSuccessful);
+    //generate steam
+    void generateSteam(bool tubesReachedNeededTemp);
+    //produce electricity
+    void produceElectricity(bool isGeneratorReceivingSteam);
+};
 /*
 Thing 5) Airplane
 5 properties:
-    1) Number of passenger seats (int)
-    2) Total weight of plane (int)
-    3) Length of plane wings (float)
-    4) Number of luggage pieces (int)
-    5) Amount of fuel (int)
+    1) number of passenger seats (int)
+    2) total weight of plane (lbs.) (float)
+    3) length of plane wings (meters) (float)
+    4) number of luggage pieces (int)
+    5) amount of fuel (gallons) (int)
 3 things it can do:
-    1) Take off and fly
-    2) Land
-    3) Send data to control tower
+    1) take off and fly
+    2) land 
+    3) send data to control tower
  */
+struct Airplane
+{
+    //number of passenger seats (int)
+    int numOfPassengerSeats = 180;
+    //total weight of plane (lbs.) (float)
+    float totalWeightOfPlane = 92354.67f;
+    //length of plane wings (meters) (float)
+    float lengthOfPlaneWings = 33.5f;
+    //number of luggage pieces (int)
+    int numOfLuggagePieces = 225;
+    //amount of fuel (gallons) (int)
+    int amountOfFuel = 20000;
+
+    struct Cockpit
+    {
+    int numOfSwitches = 30;
+    int numOfDisplays = 8;
+    int numOfFlightControllers = 2;
+    int numOfRudderPedals = 4;
+    int numOfPilots = 2;
+
+    void turnPlane(int degreesTurned);
+    void landingGearRaised(bool isGearLeverInUpPostion);
+    void talkToAirTrafficControl(bool isPushToTalkSwitchPressed);
```

Comment:
Code inside of curly braces needs to be indented 4 spaces from those curly braces. 
- [ ] revise